templates/traefik.yaml

```diff
@@ -36,6 +36,10 @@ spec:
         enabled: true
         enforced: true
         generateTLS: true
+        # dex service is exposed with TLS certificate signed by self signed root
+        # Dex CA certificate.
+        # TODO(mh): Figure out if we can configure CA certs per endpoint/ingress
```

Comment:
Is this another place it would be good to have a JIRA created for follow up (and linked here in the comments)?

Reply:
I've created https://jira.mesosphere.com/browse/DCOS-56033